### PR TITLE
Switch Arkouda graphs to display nightly by default

### PR DESCRIPTION
**Pull request by bmcdonald3**
Today, Arkouda graphs show the release version by default, which means that changes to Chapel will be missed. Switching this to default to nightly will make it more obvious of Chapel-side changes that impact Arkouda.

### CODE REVIEW

File: util/cron/common-arkouda.bash [hunk 1]
```diff
@@ -48,7 +48,7 @@ export CHPL_WHICH_RELEASE_FOR_ARKOUDA="1.30.0"
 # test against Chapel release (checking out current test/cron directories)
 function test_release() {
   export CHPL_TEST_PERF_DESCRIPTION=release
-  export CHPL_TEST_PERF_CONFIGS="release:v,nightly"
```
Comment (ronawho) [inline util/cron/common-arkouda.bash:51]: This makes "release" not visible by default. See `--configs` option in `./util/test/genGraphs --help`

Comment (bmcdonald3) [reply util/cron/common-arkouda.bash:51]: Were you thinking that we would have both of them visible by default? I was thinking we only wanted to look at nightly by default.

Comment (ronawho) [reply util/cron/common-arkouda.bash:51]: I was thinking both, but let's discuss in perf meeting today

Comment (bmcdonald3) [reply util/cron/common-arkouda.bash:51]: Switched to that, that's fine with me.

**Comment by bradcray** [comment]
Yay!

**Pull request merged**